.golangci.yml

```diff
@@ -1,23 +1,36 @@
+version: "2"
 run:
   build-tags:
     - build
     - containers_image_openpgp
-  timeout: 10m
```

Comment:
can we keep this? this avoids some timeouts when running in github actions for example. Or is this timeout already longer with v2?

Reply:
Yeah I thought this was weird that the migration tool removed it.  If I go look at their example reference config [here](https://github.com/golangci/golangci-lint/blob/544018f712a7506c49ecdb75b8cb5a4e7bbb7534/.golangci.reference.yml#L4182) it is still a valid option.  

I'll add it back in.